cmd/s2i/main.go

```diff
@@ -212,7 +212,8 @@ func newCmdRebuild(cfg *api.Config) *cobra.Command {
 				os.Exit(1)
 			}
 
-			if r, err := os.Open(cfg.DockerCfgPath); err == nil {
+			r, err := os.Open(cfg.DockerCfgPath)
+			if err == nil {
```

Comment:
@rhcarvalho yes, correct, line 215 was/is not specifically a case of shadowing.  However, below there was a shadowing case with nearly identical code which I did resolve.  I made the equivalent change from below here as well to maintain the clear similarity...